api/GPUDevice.json

```diff
@@ -653,6 +653,46 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "deprecate_depth_bias_lines_points": {
+          "__compat": {
+            "description": "<code>depthBias</code>, <code>depthBiasSlopeScale</code>, and <code>depthBiasClamp</code> must be <code>0</code> for line and point topologies.",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "128"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
```

Comment:
What is the deprecated field used for?

Reply:
Spec said A, then they changed it to B.
We announced deprecation in Chrome 128 and removed it in Chrome 131.
I'll let you pick what is more appropriate in BCD.

Reply:
I think 131 makes more sense — when it starts to produce an error rather than the warning. I've updated it.